### DISCUSSION TITLE
Welcome to Portal-2-Community-Edition Discussions!

### DISCUSSION THREAD

**Discussion by StefanH-AT** [StrataSource/Portal-2-Community-Edition#480] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by StefanH-AT** [comment]
Hello world!

**Reply by JJL772** [reply]
The world says hello back :wave:

**Reply by StefanH-AT** [reply]
I like this so far

**Reply by JJL772** [reply]
Yea, seems to work great. Github also has support for videos now, so we could avoid discord's video upload limits.
